### PR TITLE
updated Dockerfile and README

### PR DESCRIPTION
**Pull request by treyx**
updated codeowners
updated dockerfile
updated readme with SSL cert generation instructions for SMTP relay

### CODE REVIEW

**Comment by javs-perez** [comment]
Hey @treyx, I think the Readme update is missing.

File: smtp-relay-automation.sh [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
-  --overwrite
 
-aws ssm put-parameter \
-  --region us-west-2 \
-  --name "smtp-relay-key-${ENVIRONMENT}" \
-  --type "String" \
-  --value "$(cat acme.sh/smtp-relay.${DOMAIN}/smtp-relay.${DOMAIN}.key)" \
-  --overwrite
\ No newline at end of file
+# aws ssm put-parameter \
```
Comment (jacobbat) [inline smtp-relay-automation.sh:12]: Should we delete the commented out sections? We already have the history in git if we want to revert the changes...

**Review by jacobbat — APPROVED** [review]
